src/Morphir/Elm/Frontend/Resolve.elm

```diff
@@ -226,6 +227,7 @@ moduleMapping =
         , ( [ [ "string" ] ], sdkModule [ "string" ] )
         , ( [ [ "char" ] ], sdkModule [ "char" ] )
         , ( [ [ "tuple" ] ], sdkModule [ "tuple" ] )
+        , ( [ [ "rule" ] ], sdkModule [ "rule" ] )
```

Comment:
You shouldn't need this either. This is to map Elm names to Morphir names but since Rule is Morphir specific it will already have the right fully-qualified name.